lua/entities/gmod_tardis_interior/modules/sh_interior_lights.lua

```diff
@@ -61,144 +161,270 @@ if CLIENT then
         end
     end)
 
-    function ENT:AddRoundThing(pos)
-        self.roundthings[pos]=util.GetPixelVisibleHandle()
-    end
+    ENT:AddHook("ShouldDrawLight", "enabled_check", function(self,id,light)
```

Comment:
Hook should be named after the file it's in (and something else if there's multiple of the same hook in the same file)

Reply:
Should be better now